src/ros-vision-interaction/scripts/demo-interaction/sar_demo.py

```diff
@@ -65,8 +65,8 @@ def run_once(self):
             database_manager=self._database,
             interface=self._interface
         )
-        while not self._is_start_interaction:
-            rospy.sleep(1)
+        # while not self._is_start_interaction:
```

Comment:
I prefer deleting lines rather than commenting them. If we need them back, we can always look in the history.